lib/Sapi.php

```diff
@@ -67,7 +67,21 @@ static function sendResponse(ResponseInterface $response) {
             }
 
         }
-        file_put_contents('php://output', $response->getBody());
+
+        $body = $response->getBody();
+        if(is_null($body)) return;
+
+        $contentLength = $response->getHeader('Content-Length');
```

Comment:
I remember some problems in the firefox download manager and the content-length header because of a lot of http-clients using it wrong... searching for sources...